components/sections/decasteljau/content.zh-CN.md

```diff
@@ -0,0 +1,61 @@
+# de Casteljau 算法
+
+要计算贝塞尔曲线，我们必须遍历 `t` 的所有值，从 `0` 到 `1`，计算权重函数，得到 `x/y` 的值。当曲线很复杂时，计算量变得非常巨大。利用 `de Casteljau` 算法（一种几何画法），你可以轻易地用笔和尺画出曲线。
```

Comment:
You're using backticks in a lot of places, but backticks are certainly not part of standard Chinese orthography. Please change all the strings that are currently marked up with backticks to use reference quoting instead (whatever the convention for that is in mainland Chinese).